pkg/devfile/parser/data/v2/commands_test.go

```diff
@@ -355,7 +379,15 @@ func TestDevfile200_UpdateCommands(t *testing.T) {
 				},
 			}
 
-			d.UpdateCommand(tt.newCommand)
+			err := d.UpdateCommand(tt.newCommand)
+			// Unexpected error
+			if (err != nil) != tt.wantErr {
+				t.Errorf("TestDevfile200_UpdateCommands() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if err != nil {
+				return
+			}
```

Comment:
I prefer if we put the rest of the content in a `else if err == nil {}`

if you agree, we can also do it for the other tests in this PR

Reply:
updated